src/lib/modules/blockchain_connector/index.js

```diff
@@ -107,10 +107,14 @@ class BlockchainConnector {
 
     if (type === 'vm') {
       const sim = self._getSimulator();
-      const options = Object.assign({}, self.config.contractsConfig.deployment, {gasPrice: "0x01", gasLimit: "0xfffffffffffff"});
-      self.provider = sim.provider(options);
+      const options = Object.assign({}, self.config.contractsConfig.deployment, {
+        gasPrice: "0x01",
+        gasLimit: "0xfffffffffffff"
+      });
 
       if (coverage) {
+        options.allowUnlimitedContractSize = true;
```

Comment:
Instead of having two lines for `self.provider = sim.provider(options);`, you could just put on line 113
```
  allowUnlimitedContractSize: coverage
});
self.provider = sim.provider(options);
```

Reply:
done